README.md

```diff
@@ -152,6 +152,78 @@ Add the following settings to your `.vscode/settings.json`:
 }
 ```
 
+## Neovim Support
+
+```lua
+local customizations = {
+  { rule = 'style/*', severity = 'off', fixable = true },
+  { rule = 'format/*', severity = 'off', fixable = true },
+  { rule = '*-indent', severity = 'off', fixable = true },
+  { rule = '*-spacing', severity = 'off', fixable = true },
+  { rule = '*-spaces', severity = 'off', fixable = true },
+  { rule = '*-order', severity = 'off', fixable = true },
+  { rule = '*-dangle', severity = 'off', fixable = true },
+  { rule = '*-newline', severity = 'off', fixable = true },
+  { rule = '*quotes', severity = 'off', fixable = true },
+  { rule = '*semi', severity = 'off', fixable = true },
+}
+
+local lspconfig = require('lspconfig')
+-- Enable eslint for all supported languages
+lspconfig.eslint.setup(
+  {
+    filetypes = {
+      "javascript",
```

Comment:
Should we also add all the default filetypes?

```vim
lua =require("lspconfig.server_configurations.eslint").default_config.filetypes
" { "javascript", "javascriptreact", "javascript.jsx", "typescript", "typescriptreact", "typescript.tsx", "vue", "svelte", "astro" }
```

We're only missing these two: `javascript.jsx` and `typescript.tsx`.

Reply:
`svelte` and `yaml` are missing too.

Reply:
Hi @thenbe Thanks for your review, as someone who is not doing a lot react. Can I ask why there's two filetype for `tsx`? E.g. The filetype I get from a `tsx` file is `typescriptreact`. Btw yaml is there.

![image](https://github.com/user-attachments/assets/051a9d6e-cc60-41f0-b158-27f8495b86f9)

Reply:
Seems like they're included in nvim-lspconfig because both variants are widely used: https://github.com/neovim/nvim-lspconfig/pull/2419#discussion_r1085484805. FWIW I also get `typescriptreact`.

Reply:
Interesting, I have added these two and I would guess it's a legacy issue ( since I cannot find a clear explanation )?

Reply:
Yeah that's pretty much the same conclusion I reached, that they're included for legacy reasons.